Documentation/user-guides/cluster-monitoring.md

```diff
@@ -0,0 +1,394 @@
+# Cluster Monitoring
+
+This guide is intended to give an introduction to all the parts required to start monitoring a Kubernetes cluster with Prometheus using the Prometheus Operator.
+
+This guide assumes you have a basic understanding of how to use the functionality the Prometheus Operator implements. If you haven't yet we recommend reading through the [getting started guide](getting-started.md) as well as the [alerting guide](alerting.md).
+
+## Metric Sources
+
+Monitoring a Kubernetes cluster with Prometheus is a natural as Kubernetes components themselves are instrumented with Prometheus metrics, therefore those components simply have to be discovered by Prometheus and most of the cluster is monitored.
+
+Metrics that are rather about cluster state than a single components metric is exposed by the add-on component [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics).
+
+Additionally to have an overview of cluster nodes resources the Prometheus [node_exporter](https://github.com/prometheus/node_exporter) is used. The node_exporter allows monitoring a node's resources from CPU, memory and disk utilization and more.
```

Comment:
Additionally,

nodes'

"resources from CPU, memory and..." – the "from"  doesn't make sense here.